www/layouts/shortcodes/x-keyboard.html

```diff
@@ -1,5 +1,18 @@
 {{ $name := .Get "name" }}
 <div class="keyboard">
+  {{ if $name }}
```

Comment:
This change was aiming at moving the menu/links above the keyboard (instead of below the keyboard). Idem, feel free to reject that obviously.